api/admin/controllers/user.js

```diff
@@ -170,7 +175,7 @@ module.exports = wrapHandlers({
       link,
     });
     if (link) {
-      await Mailer.sendUAAInvite(email, link);
+      await Mailer.sendUAAInvite(email, link, origin, org.name);
```

Comment:
oooh does this mean i could use the origin domain and/or the org name in my invite email???